### ISSUE TITLE
Feature Request: Trim prefabs from closed maps

### ISSUE THREAD

**Issue by S34NW** [SpaiR/StrongDMM#145]
### Problem to be solved

When you open a map and then close it, the prefabs/instances in use on that map remain in the list.
This can sometimes be frustrating if there are many var-edited items you do not want to see from other maps, as the only way to clear the list currently is to reload the whole environment.

### Sugest a solution

A preference or button to rebuild the list based on currently open maps might work, but it should not be forced as I can imagine some users wish these objects to remain present for ease of placement.

**Comment by SpaiR** [comment]
Yeah, thats a pretty good stuff to add. I've thought about that in the same way: button to clear unused instances. I imagine it as a part of the overall instances panel improvements.
